SDVModTest/UIElements/ShowItemHoverInformation.cs

```diff
@@ -102,44 +102,47 @@ private void OnInventoryChanged(object sender, InventoryChangedEventArgs e)
         private void PopulateRequiredBundles()
         {
             _prunedRequiredBundles.Clear();
-            foreach (var bundle in _bundleData)
+            if (!_communityCenter.areAllAreasComplete() && !Game1.player.mailReceived.Contains("JojaMember"))
             {
-                String[] bundleRoomInfo = bundle.Key.Split('/');
-                String bundleRoom = bundleRoomInfo[0];
-                int roomNum;
-
-                switch(bundleRoom)
-                {
-                    case "Pantry": roomNum = 0; break;
-                    case "Crafts Room": roomNum = 1; break;
-                    case "Fish Tank": roomNum = 2; break;
-                    case "Boiler Room": roomNum = 3; break;
-                    case "Vault": roomNum = 4; break;
-                    case "Bulletin Board": roomNum = 5; break;
-                    default: continue;
-                }
-
-                if (_communityCenter.shouldNoteAppearInArea(roomNum))
-                {
-                    int bundleNumber = bundleRoomInfo[1].SafeParseInt32();
-                    string[] bundleInfo = bundle.Value.Split('/');
-                    string bundleName = bundleInfo[0];
-                    string[] bundleValues = bundleInfo[2].Split(' ');
-                    List<int> source = new List<int>();
-
-                    for (int i = 0; i < bundleValues.Length; i += 3)
-                    {
-                        int bundleValue = bundleValues[i].SafeParseInt32();
-                        if (bundleValue != -1 &&
-                            !_communityCenter.bundles[bundleNumber][i / 3])
-                        {
-                            source.Add(bundleValue);
-                        }
-                    }
-
-                    if (source.Count > 0)
-                        _prunedRequiredBundles.Add(bundleName, source);
-                }
+	            foreach (var bundle in _bundleData)
+	            {
+		            String[] bundleRoomInfo = bundle.Key.Split('/');

```

Comment:
Your tabbing seems to be set to 8. Please keep it at the 4 my project is using.

Reply:
Hm weird my usual default is 4 but I will look into it in a bit.

Reply:
Okay the problem is I did not replace tabs with spaces and github displays tabs as 8 spaces. (See: https://github.com/github/VisualStudio/issues/1524)
I will go through all my changes and replace all tabs with spaces.